src/main/java/org/folio/rest/core/RestClient.java

```diff
@@ -18,60 +18,54 @@
 
 public class RestClient {
 
-    private static final Logger logger = LogManager.getLogger();
-    private static final String EXCEPTION_CALLING_ENDPOINT_MSG = "Exception calling %s %s - %s";
-    public static final String OKAPI_URL = "x-okapi-url";
+  private static final Logger log = LogManager.getLogger();
+  private static final String EXCEPTION_CALLING_ENDPOINT_MSG = "Exception calling %s %s - %s";
+  public static final String OKAPI_URL = "x-okapi-url";
 
 
-    public <T> CompletableFuture<T> getById(String baseEndpoint, String id, RequestContext requestContext, Class<T> responseType) {
-        RequestEntry requestEntry = new RequestEntry(baseEndpoint).withPathParameter("id", id);
-        return get(requestEntry, requestContext, responseType);
-    }
+  public <T> CompletableFuture<T> getById(String baseEndpoint, String id, RequestContext requestContext, Class<T> responseType) {
+    RequestEntry requestEntry = new RequestEntry(baseEndpoint).withPathParameter("id", id);
+    return get(requestEntry, requestContext, responseType);
+  }
 
-    public <S> CompletableFuture<S> get(RequestEntry requestEntry, RequestContext requestContext, Class<S> responseType) {
-        CompletableFuture<S> future = new CompletableFuture<>();
-        String endpoint = requestEntry.buildEndpoint();
-        HttpClientInterface client = getHttpClient(requestContext.getHeaders());
-        if (logger.isDebugEnabled()) {
-            logger.debug("Calling GET {}", endpoint);
-        }
+  public <S> CompletableFuture<S> get(RequestEntry requestEntry, RequestContext requestContext, Class<S> responseType) {
+    CompletableFuture<S> future = new CompletableFuture<>();
+    String endpoint = requestEntry.buildEndpoint();
+    HttpClientInterface client = getHttpClient(requestContext.getHeaders());
+    log.debug("Calling GET {}", endpoint);
```

Comment:
We need to create separate story to delete this deprecated http client, I will create